src/Mobile.Bff.Shopping/Program.cs

```diff
@@ -3,36 +3,15 @@
 builder.AddServiceDefaults();
 builder.AddApplicationServices();
 
+builder.Services.AddReverseProxy()
+    .LoadFromConfig(builder.Configuration.GetSection("ReverseProxy"));
+
 var app = builder.Build();
 
 app.UseHttpsRedirection();
 
 app.MapDefaultEndpoints();
 
-const string CatalogApiPrefix = "/api/v1/catalog/";
-var forwardedCatalogApis = new[]
-{
-    CatalogApiPrefix + "items",
-    CatalogApiPrefix + "items/by",
-    CatalogApiPrefix + "items/{id}",
-    CatalogApiPrefix + "items/by/{name}",
-
-    CatalogApiPrefix + "items/withsemanticrelevance/{text}",
-
-    CatalogApiPrefix + "items/type/{typeId}/brand/{brandId?}",
-    CatalogApiPrefix + "items/type/all/brand/{brandId?}",
-    CatalogApiPrefix + "catalogTypes",
-    CatalogApiPrefix + "catalogBrands",
-
-    CatalogApiPrefix + "items/{id}/pic",
-};
-
-foreach (var forwardedUrl in forwardedCatalogApis)
-{
-    var mapFromPattern = "/catalog-api" + forwardedUrl;
```

Comment:
The only thing I don't see in the proxy config now is the `/catalog-api` prefix that I think is used for the 'local name' of the API. That is, the Mobile App (e.g. .NET MAUI Blazor Hybrid app) talk to the Mobile BFF at:

```
http://mobile-bff/catalog-api/<some catalog path>
                  ^^^^^^^^^^^
```

But I don't see where the new proxy config has that? Or is it not needed?

Ultimately just running the .NET MAUI Blazor Hybrid app (the `HybridApp` folder in the repo) will either work or not work, but my inclination is that it won't work because this path doesn't exist. Unless the proxy config just figures that out somehow?

Reply:
hmmmm let me take a look, I thought this would do it - "Address": "http://catalog-api/api/v1/catalog""